ste/mgr-JobPartMgr.go

```diff
@@ -355,7 +364,7 @@ func (jpm *jobPartMgr) ScheduleTransfers(jobCtx context.Context) {
 			// If source doesn't exists, skip the transfer
 			_, ok := includeTransfer[src]
 			if !ok {
-				jpm.ReportTransferDone() // Don't schedule transfer which is not mentioned to be included
+				jpm.ReportTransferDone(ts) // Don't schedule transfer which is not mentioned to be included
```

Comment:
I'm not sure we actually _use_ includeTransfer and exclude transfer.

@zezha-msft can you confirm whether we do?

@nakulkar-msft If if turns out that we don't use those, maybe you could just panic here, saying that "includeTransfer" lists are unsupported. (And the same below for exclude below).

The reasons I suggest this are that:
(a) in these cases, I would expect `ts` to be `Started` (the default value) for these, and that will get a panic in ReportTransferDone.
(b) Having these exclude and include lists is icky in general, and I think even more icky because they don't cleanly fit into any of the statues that you're looking at.

(I guess an alternative might be to report them done as "skipped"??? Maybe see what Ze thinks.  As noted above, I'm not sure the code in question every even gets called these days.  It was an old idea, and I'm not sure we still support it.  Maybe one day in the future, not in this PR, we could remove the code completely IF its obsolete.)

Reply:
This code is indeed obsolete. Let's remove it.

---
In reply to: [448049859](https://github.com/Azure/azure-storage-azcopy/pull/1058#discussion_r448049859) [](ancestors = 448049859)

Reply:
With confirmation that is Obsolete, I'll 'panic' if we reach those statements.